CHANGELOG.rst

```diff
@@ -14,6 +14,11 @@ Change Log
 Unreleased
 ----------
 
+[0.52.1] - 2017-10-17
+---------------------
+
+* Added EnterpriseCustomerCatalog UUID as query parameter on enrollment URLs.
```

Comment:
Add EnterpriseCustomerCatalog UUID as query parameter  `"catalog"` in enterprise course and program enrollment URL's.

Reply:
In the future, let's not add this or the version bump until final PR approval. It will make merging to the Sprint branch a bit easier.